packages/telemetry/src/system-info.ts

```diff
@@ -52,7 +52,8 @@ export function getSystemInfo(versions: { viteVersion: string; astroVersion: str
 	}
 
 	const cpus = os.cpus() || [];
-	meta = {
+
+	return {
```

Comment:
I think there's a bug here. We want to assign to `meta` so that it is cached on the second run. The new code doesn't assign to it anymore

Reply:
Ah, this is what I get for hand-editing 😅 good catch. Will undo.